webpack.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = (env, argv) => {
     const config = webpackConfig(env, argv);
 
     // For adding custom admin js file uncomment the following line:
-    // config.entry.unshift(path.resolve(__dirname, 'assets/admin/index.js'));
+    // config.entry.push(path.resolve(__dirname, 'assets/admin/index.js'));
```

Comment:
this does not fix it correctly in all cases as the [entriesCount](https://github.com/sulu/sulu/blob/6766a36cc2f0c807c59202bf7540a08c4ef31192/webpack.config.js#L36)  variable is not correct. /cc @danrot 

An idea would be instead of extending the config do something like:

```
env.suluEntries = [];
env.suluEntries.push(path.resolve(__dirname, 'assets/admin/index.js'));

return webpackConfig(env, argv);
```

and in vendor/sulu/sulu/webpack.config.json:

```json
    const entries = env.bundles ? env.bundles : [];
```